src/plugins/rename/rename.c

```diff
@@ -207,11 +234,6 @@ static Key * restoreKeyName (Key * key, const Key * parentKey, const Key * confi
 			Key * result = keyDup (key);
 			keySetName (result, keyString (origNameKey));
 			keySetMeta (result, ELEKTRA_ORIGINAL_NAME_META, 0);
-
-			if (!hasSync)
```

Comment:
Still removed. Could you also please add a comment that it is here because of #404. (At all places)